static/app/types/debugFiles.tsx

```diff
@@ -81,10 +81,10 @@ export type CustomRepoAppStoreConnect = {
   appName: string;
   appconnectIssuer: string;
   appconnectKey: string;
-  appconnectPrivateKey: string;
   bundleId: string;
   id: string;
   name: string;
+  appconnectPrivateKey: {'hidden-secret': boolean} | string;
```

Comment:
Should we make this a new type alias?

Reply:
sure thing...and I just doubled checked and the string is not really needed